agent-bootstrap/src/main/java/io/opentelemetry/auto/config/Config.java

```diff
@@ -96,6 +96,9 @@
   private static final String DEFAULT_TRACE_EXECUTORS = "";
   private static final String DEFAULT_TRACE_METHODS = null;
 
+  public static final String DISABLE_SQL_NORMALIZER = "disable.sql.normalizer";
+  public static final boolean DEFAULT_DISABLE_SQL_NORMALIZER = false;
```

Comment:
I hear you about naming `boolean` settings so that the default maps to `false`, but I think this naming would be more consistent with the existing settings:

```suggestion
  public static final String SQL_NORMALIZER_ENABLED = "sql.normalizer.enabled";
  public static final boolean DEFAULT_SQL_NORMALIZER_ENABLED = true;
```

Reply:
Agreed, will turn into another commit to fix the test though 😄